src/licensedcode/data/rules/apache-2.0_or_bsd-new_8.RULE

```diff
@@ -1,11 +1,11 @@
 ---
 license_expression: apache-2.0 OR bsd-new
 is_license_notice: yes
-relevance: 99
+relevance: 100
```

Comment:
Do not change this. How can you be 100% sure that LICENSE.BSD contains a bsd-new